src/components/ShareResultsModal/EmailField.tsx

```diff
@@ -42,6 +44,18 @@ export default function EmailField({ onSubmit }: EmailFieldProps) {
     }
   };
 
+  const onCloseToaster = () => {
+    setShowNotification(false);
+    setIsSubmitted(false);
+    setFormError(null);
+  };
+
+  useEffect(() => {
```

Comment:
Instead of using a `useEffect` here, would it be better to `setShowNotification` in the same place we set `isSubmitted` and `formError`?